.code-samples.meilisearch.yaml

```diff
@@ -41,12 +41,44 @@ search_post_1: |-
   client.index('movies').search('American ninja')
 search_get_1: |-
   client.index('movies').search('American ninja')
-get_update_1: |-
-  client.index('movies').getUpdateStatus(1)
-get_all_updates_1: |-
-  client.index('movies').getAllUpdateStatus()
-get_keys_1: |-
+get_task_by_index_1: |-
+  client.index('movies').getTask(1)
+get_all_tasks_by_index_1: |-
+  client.index('movies').getTasks()
+get_all_tasks_1: |-
+  client.getTasks()
+get_task_1: |-
+  client.getTask(1)
+get_one_key_1: |-
+  client.getKey(d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4)
```

Comment:
I think we should add the quotes here! `'``'` 🎃 

`client.getKey('d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4')`

Reply:
```suggestion
  client.getKey('d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4')
```